server/mailet/integration-testing/src/test/java/org/apache/james/smtp/SmtpIdentityVerificationTest.java

```diff
@@ -334,23 +334,24 @@ void shouldAcceptGroupWhenOnlyUser(@TempDir File temporaryFolder) throws Excepti
     }
 
     @Test
-    void errorsShouldBeIgnoredWhenUnAuthed(@TempDir File temporaryFolder) throws Exception {
```

Comment:
Removed this test as the From header was empty. 

I think `shouldAcceptInvalidFromWhenUnauthenticated` test with from: INVALID covers this?